src/provisioner/const.py

```diff
@@ -20,3 +20,5 @@
 DEFAULT_LOG_LEVEL = "INFO"
 SUPPORTED_LOG_LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
 RELEASE_INFO_URL = "yaml:///opt/seagate/cortx/RELEASE.INFO"
+UPGRADE_MODE_KEY = "UPGRADE_MODE"
+UPGRADE_MODE = ""
```

Comment:
Make this:
`UPGRADE_MODE_KEY`
`UPGRADE_MODE_VAL`

Reply:
done